lxd/resource_lxd_container.go

```diff
@@ -174,6 +180,8 @@ func resourceLxdContainerCreate(d *schema.ResourceData, meta interface{}) error
 		image = imgParts[1]
 	}
 	config := resourceLxdConfigMap(d.Get("config"))
+	config = resourceLxdConfigMapAppend(d.Get("limits"), config, "limits.")
```

Comment:
So between the above two lines of code, if someone specified `limits.cpu` in the `config` block and then specified `cpu` in the `limits` block, the latter should overwrite the former? If so 👍